lib/ota/zigbeeOTA.js

```diff
@@ -11,13 +11,15 @@ async function getImageMeta(current, logger, device) {
     const modelId = device.modelID;
     const imageType = current.imageType;
     const manufacturerCode = current.manufacturerCode;
+    const manufacturerName = current.manufacturerName;
```

Comment:
I think this should be `const manufacturerName = device.manufacturerName;`

Reply:
Yes